docs/templates/template_reference.md

```diff
@@ -2,17 +2,24 @@
 
 #### accounts_password
 -   Checks if PAM enforces password quality requirements. Checks the
-    configuration in `/etc/pam.d/system-auth` (for RHEL 6 systems) or
-    `/etc/security/pwquality.conf` (on other systems).
+    configuration in `/etc/security/pwquality.conf`.
```

Comment:
Maybe we should add to the template description that it only compares the parameter value against the xccdf variable, that it does not check the value of the parameter in relation to zero.
And to compare the parameter value against zero, `zero_boundary_operation` needs to be set.